no-pdi/include/pdi.h

```diff
@@ -0,0 +1,299 @@
+/*******************************************************************************
+* Copyright (C) 2025 Commissariat a l'energie atomique et aux energies alternatives (CEA)
+* All rights reserved.
+*
+* Redistribution and use in source and binary forms, with or without
+* modification, are permitted provided that the following conditions are met:
+* * Redistributions of source code must retain the above copyright
+*   notice, this list of conditions and the following disclaimer.
+* * Redistributions in binary form must reproduce the above copyright
+*   notice, this list of conditions and the following disclaimer in the
+*   documentation and/or other materials provided with the distribution.
+* * Neither the name of CEA nor the names of its contributors may be used to
+*   endorse or promote products derived from this software without specific
+*   prior written permission.
+*
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+* IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+* AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+* LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+* OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+* THE SOFTWARE.
+******************************************************************************/
+
+/** \file pdi.h for deactivation
+ *
+ * C user API
+ *
+ * The user facing API is the interface offered by PDI to C application
+ * developers.
+ *
+ * \defgroup init_final Initialization and finalization
+ *
+ * The initialization and finalization part of the API is used to setup PDI,
+ * release its resources and check version information.
+ *
+ * \defgroup annotation Code annotation
+ *
+ * The code annotation API is the main interface to use in the code.
+ *
+ * It offers functions that can be called from code with no side effect by
+ * default and that can therefore be considered as annotations.
+ *
+ * \defgroup error Error handling
+ *
+ * The error handling API supports checking the error status of PDI.
+ *
+ * By default, errors in PDI C API are signaled by a return code of type
+ * PDI_status_t and an error message can be retrieved with the PDI_errmsg
+ * function. This default behavior can be changed by replacing the error handler
+ * with the PDI_errhandler function.
+ *
+ */
+
+#ifndef PDI_H_
+#define PDI_H_
+
+// struct PC_tree_t;
+
+#define PDI_EXPORT __attribute__((visibility("default")))
```

Comment:
all `PDI_EXPORT` and such come from `config.h` that is not generated here. Please remove them